README.md

```diff
@@ -23,23 +23,34 @@ Connecting those modules with each other lets you create immensly useful agents.
 
 ## Installation
 
-Beehive requires Go 1.8 or higher. Make sure you have a working Go environment. See the [install instructions](http://golang.org/doc/install.html).
+Beehive requires Go 1.8 or higher. Make sure you have a working Go environment.
+See the [installation instructions of Go](http://golang.org/doc/install.html).
 
 ### From source
 
-The recommended way is to fetch the sources and run make. This requires [dep](https://github.com/golang/dep)
-to be installed on your system.
+The recommended way is to fetch the sources and run make.
+This requires [dep](https://github.com/golang/dep) to be installed on your system.
 
+If the `$GOPATH` environment variable is empty (`echo $GOPATH`), set it with:
+    
+    # Set the $GOPATH environment variable for the active shell
+    export GOPATH="/your/go/path"
+
+Create the directory tree `$GOPATH/src/github.com/muesli/` and run this script:
+
+    cd $GOPATH/src/github.com/muesli/
+    # GNU Git should download the sources in the beehive/ directory
     git clone https://github.com/muesli/beehive.git
     cd beehive
     make get-deps
```

Comment:
Before `make get-deps` we should probably add a `git submodule update --init` just to be safe.

Reply:
@muesli the Makefile already contains:
```
submodule:
     git submodule update --init
```
It is already automatically executed with the simple "make" command, in order before the other commands. If you think it's a necessary command then add it, but more commands are always equivalent to less users willing to run them.

Reply:
You're right!

Reply:
@muesli the push request #209 completes the facilitation work by providing a clearer and more friendly installer. In this way even if they (the end users) have read the documentation in a superficial way they can understand all the changes made to the system.
I used simple syntaxes to allow anyone to change the installation process over time. In addition I have also added the classic "contributors" file to attract other developers. Obviously you will have to manually give approval for this commit too.